### DISCUSSION TITLE
Is there a way to change the default scratchpads height and width?

### DISCUSSION THREAD

**Discussion by Muhammad95959** [i3/i3#6044] [Q&A] [answered]
I'm looking for something like this
```
for_window [class="scratchpad"] resize set 1280px 720px
```

**Comment by okraits** [comment, marked as answer]
How about:

`bindsym ... floating enable,resize set ...,move scratchpad`

**Reply by Muhammad95959** [reply]
Thanks for the answer that worked for me after a simple tweak to be able to use only one keybinding for moving and showing the scratchpad
```
bindsym $mod+s exec i3-msg scratchpad show || bash -c "i3-msg floating enable && i3-msg resize set 1280px 720px && i3-msg move position center && i3-msg move scratchpad"
```

**Reply by okraits** [reply]
You don't need that complicated command. You just need the command I posted. Of course you need to replace the `...`with the actual keysyms and the desired geometry. Then you can use `scratchpad show`as usual.

**Reply by Muhammad95959** [reply]
That needs a second keybinding for ```scratchpad show``` if I understood correctly and i don't want more than one for handling the scratchpad

**Reply by okraits** [reply]
Of course, because moving a window into the scratchpad and showing the scratchpad is done by two separate commands.

**Reply by Muhammad95959** [reply]
> ```
> bindsym $mod+s exec i3-msg scratchpad show || bash -c "i3-msg floating enable && i3-msg resize set 1280px 720px && i3-msg move position center && i3-msg move scratchpad"
> ```

The previous command does both using one keybinding 
if there's no scratchpads it executes ```bash -c "$code"``` part and if not it executes ```scratchpad show```
